src/main/java/mods/eln/sixnode/tutorialsign/TutorialSignElement.java

```diff
@@ -53,51 +54,54 @@ public static String getText(String balise) {
 				int i = 0;
 			} catch (Exception e) {
 			}
-*/
+            */
             //optional, but recommended
             //read this - http://stackoverflow.com/questions/13786607/normalization-in-dom-parsing-with-java-how-does-it-work
             //doc.getDocumentElement().normalize();
-
-            try {
-                String file = Utils.readMapFile("EA/tutorialSign.txt");
-                String ret;
-                if (file.contains("\r\n"))
-                    ret = "\r\n";
-                else
-                    ret = "\n";
-
-                file = file.replaceAll("#" + ret, "#");
-                file = file.replaceAll(ret + "#", "#");
-
-                String[] split = file.split("#");
-
-                boolean first = true;
-                int counter = 0;
-                String baliseTag = "";
-
-                for (String str : split) {
-                    if (first) {
-                        first = false;
-                        continue;
-                    }
-                    if (counter == 0) {
-                        baliseTag = str;
-                    }
-                    if (counter == 1) {
-                        baliseMap.put(baliseTag, str);
-                    }
-
-                    counter = (counter + 1) & 1;
-                }
-            } catch (IOException e) {
-                //	e.printStackTrace();
-            }
+            String localizedPath = "EA/tutorialSign/" + getCurrentLanguage() + ".txt"; //localized file path
+            String defaultPath = "EA/tutorialSign/en_US.txt"; //if the localized file is not found, then load the default file (en_US)
```

Comment:
Nitpick - please use `.lang` extension and the format used in the language files. Exception permitted for the backwards compat.